guides/common/modules/proc_assigning-content-view-environments-to-activation-keys.adoc

```diff
@@ -0,0 +1,38 @@
+[id="assigning-content-view-environments-to-activation-keys"]
+= Assigning content view environments to activation keys
+
+You can assign multiple content view environments to an activation key to provide hosts with access to content from each assigned environment.
+Hosts registered with the activation key receive the environments in the order they are stored.
+For more information, see xref:content-view-environment-ordering-and-priority[].
+
+.Prerequisites
+* The *Allow multiple content views* setting must be enabled.
+For more information on content settings, see {AdministeringDocURL}content_settings_{context}[Content settings] in _{AdministeringDocTitle}_.
+
+.CLI procedure
+. Display the available content view environments to obtain their labels or IDs:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view-environment list
+----
++
+. Assign content view environments to an activation key using their labels:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update \
+--id _My_Activation_Key_ID_
+--organization-id _My_Organization_ID_ \
+--content-view-environments _"My_env1/My_cv1,My_env2/My_cv2"_
+----
++
+. Assign content view environments to an activation key using their IDs:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update \
+--id _My_Activation_Key_ID_
+--organization-id _My_Organization_ID_ \
+--content-view-environment-ids [_My_content_view_env_ID_1_, _My_content_view_env_ID_2_] 
+----
```

Comment:
```suggestion
. Assign content view environments to an activation key:
* By using their labels:
+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key update \
--id _My_Activation_Key_ID_ \
--organization-id _My_Organization_ID_ \
--content-view-environments _"My_env1/My_cv1,My_env2/My_cv2"_
----
* By using their IDs:
+
[options="nowrap" subs="+quotes"]
----
$ hammer activation-key update \
--id _My_Activation_Key_ID_ \
--organization-id _My_Organization_ID_ \
--content-view-environment-ids [_My_content_view_env_ID_1_, _My_content_view_env_ID_2_] 
----
```
Again, these are two options of the same action.
Note that the `--id` lines were missing `\` at the end.